Makefile

```diff
@@ -341,7 +341,7 @@ beautify: virtualenv/bin/fixjsstyle
 spritegen: $(SPRITE_GENERATED_FILES)
 
 $(COMPILED_JSX_FILES): $(JSX_FILES)
-	jsx --no-cache-dir -x jsx . .
+	# jsx --no-cache-dir -x jsx . .
```

Comment:
This code isn't going to stay around at all correct? Shouldn't it just get
removed?
On Apr 23, 2015 12:06 PM, "Jeff Pihach" <notifications@github.com> wrote:

> In Makefile
> <https://github.com/juju/juju-gui/pull/719#discussion_r28977107>:
>
> > @@ -341,7 +341,7 @@ beautify: virtualenv/bin/fixjsstyle
> >  spritegen: $(SPRITE_GENERATED_FILES)
> >
> >  $(COMPILED_JSX_FILES): $(JSX_FILES)
> > -	jsx --no-cache-dir -x jsx . .
> > +	# jsx --no-cache-dir -x jsx . .
>
> Driveby - this implementation was never finished and causes issues when
> running tests so it's been commented out until we can finish it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/juju-gui/pull/719/files#r28977107>.
>